build.sh

```diff
@@ -303,6 +299,16 @@ case $OSName in
         __TestNugetRuntimeId=osx.10.10-x64
         ;;
 
+    Linux)
+        __BuildOS=Linux
+        ;;
+
+    NetBSD)
+        __BuildOS=NetBSD
+        __ToolNugetRuntimeId=osx.10.10-x64
```

Comment:
I see that you have just replicated the FreeBSD snippet, but osx is not going to work well for FreeBSD/NetBSD....